content-src/asrouter/components/SnippetBase/SnippetBase.jsx

```diff
@@ -30,7 +30,7 @@ export class SnippetBase extends React.PureComponent {
     }
 
     return (
-      <button className="blockButton" title={this.props.content.block_button_text} onClick={this.onBlockClicked} />
+      <button className="blockButton" title={this.props.content.block_button_text || "Remove this"} onClick={this.onBlockClicked} />
```

Comment:
This should be localized eventually

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1500489